components/colorarea/stories/template.js

```diff
@@ -0,0 +1,36 @@
+// template is where the component markup with go
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorArea",
+  customClasses = [],
+  isDisabled = false,
+  customSize,
```

Comment:
I like the idea of breaking these out as their own types (maybe string type so user's can put in units too?) - what do you think?  `width` & `height`? We can convert the values into the variables inside the template & optionally append a px unit if none is present.

Reply:
taking a look at this! I am understanding your suggestion as instead of using custom size and having `customSize: "--mod-colorarea-height: 80px; --mod-colorarea-width: 80px;",` to use separate variables for width and height  in the story and convert this to the string we need within the template. I can see how that is more flexible.

Reply:
Ok got it implemented let me know if you have feedback on how I approached it!